app/utils/bundle-importer.js

```diff
@@ -460,7 +481,20 @@ YUI.add('bundle-importer', function(Y) {
               name = requiredModel.get('name') + '/' + size;
               break;
             case 2:
-              record.args[2] = requiredModel.id;
+              // Check whether this placement refers to a machine or a unit.
+              if (arg.indexOf('$addUnit') === 0) {
```

Comment:
Will do. (I disagree, because I mean precisely that '$addUnit' must be at position 0; but I noticed >-1 above, so I'll go with that)